libs/consent/api/src/lib/userConsent/user-consent.service.ts

```diff
@@ -0,0 +1,116 @@
+import {Injectable, NotFoundException} from '@nestjs/common'
+import {PrismaClient} from '@prisma/client'
+import {UserSession} from '@wepublish/authentication/api'
+import {
+  UserConsent,
+  UserConsentInput,
+  UpdateUserConsentInput,
+  UserConsentFilter
+} from './user-consent.model'
+
+@Injectable()
+export class UserConsentService {
+  constructor(private prisma: PrismaClient) {}
+
+  /*
+  Queries
+ */
+  async userConsentList(filter?: UserConsentFilter): Promise<UserConsent[]> {
+    const data = await this.prisma.userConsent.findMany({
+      where: {
```

Comment:
```
where: {
   value: filter.value,
        consent: {
          name: filter.name,
          slug: filter.slug
        }
      },
```

Is probably what you wanted. Currently there's a bug here where value does nothing